node/hyperbahn/service_proxy.js

```diff
@@ -84,6 +92,26 @@ function handleRequest(req, buildRes) {
         return;
     }
 
+    // apply rate limiter
+    if (req.serviceName !== 'hyperbahn') {
```

Comment:
Let's apply rate limiting for all services.

We will configure the rate for `hyperbahn`, `ringpop` & `autobahn` seperately.